packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx

```diff
@@ -55,7 +55,16 @@ interface PersonaSelectPanelProps extends withClasses<'checked' | 'unchecked' |
 
 export const PersonaSelectPanel = memo<PersonaSelectPanelProps>((props) => {
     const { finishTarget, enableVerify = true, onClose } = props
-    const { t } = useI18N()
+
+    const { t: originalTran } = useI18N()
+    const t: typeof originalTran = useCallback(
+        (key, options = {}) => {
+            if (typeof options === 'string') return t(key, options)
```

Comment:
Can we handle it in useI18N?

Reply:
@lelenei Learn unescape from https://www.i18next.com/translation-function/interpolation#unescape

Reply:
keyUnescaped does not seem to support

Reply:
Unescaped not available